Issue #2375: Add a view for managing files.

jenlampton opened the pull request:
Fixes https://github.com/backdrop/backdrop-issues/issues/2375

mikemccaffrey commented:
Hmm... why am I not seeing a view for the files in the testing site?

jenlampton commented:
Are there any files? Maybe we should add some empty text.

quicksketch commented:
Yeah the view itself seems to be missing. There's no additional default config file in the diff. Perhaps it was left out of the PR?

jenlampton commented:
> Yeah the view itself seems to be missing. 

whaaa!!!! sorry about that. Added back. I also moved all the views file stuff to file module in this PR since it seemed weird to be adding new things into system. I left file entity over there tho. We can move that in https://github.com/backdrop/backdrop-issues/issues/213

klonos commented on core/modules/file/config/views.view.file_admin.json:
```diff
@@ -0,0 +1,467 @@
+    "disabled": false,
+    "base_table": "file_managed",
+    "human_name": "Administer files",
+    "core": "1.5.x-dev",
```
Is this core version declaration required? What purpose does it serve?

jenlampton replied:
I don't know what that does, it was added by views!

jenlampton replied:
removed :)

klonos replied:
https://github.com/backdrop/backdrop-issues/issues/2432 😉

quicksketch commented:
I created a new PR replacing this one at https://github.com/backdrop/backdrop/pull/1696.
